Xenial and Kinetic Tests Failing

## Description
Both the scheduled tests for [xenial](https://github.com/ros-tooling/setup-ros/runs/796948421?check_suite_focus=true) and [kinetic](https://github.com/ros-tooling/setup-ros/runs/796948523?check_suite_focus=true) are failing.

### Expected Behavior
These tests should pass.

### Actual Behavior

*Xenial*
```
      Complete output from command python setup.py egg_info:
      Traceback (most recent call last):
        File "<string>", line 1, in <module>
        File "/tmp/pip-build-k5hdlbcq/numpy/setup.py", line 30, in <module>
          raise RuntimeError("Python version >= 3.6 required.")
      RuntimeError: Python version >= 3.6 required.
      
      ----------------------------------------
  Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-k5hdlbcq/numpy/
  You are using pip version 8.1.1, however version 20.1.1 is available.
  You should consider upgrading via the 'pip install --upgrade pip' command.
##[error]The process '/usr/bin/sudo' failed with exit code 1
```

*Kinetic*
```
      Complete output from command python setup.py egg_info:
      Traceback (most recent call last):
        File "<string>", line 1, in <module>
        File "/tmp/pip-build-70vtrzws/numpy/setup.py", line 30, in <module>
          raise RuntimeError("Python version >= 3.6 required.")
      RuntimeError: Python version >= 3.6 required.
      
      ----------------------------------------
  Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-70vtrzws/numpy/
  You are using pip version 8.1.1, however version 20.1.1 is available.
  You should consider upgrading via the 'pip install --upgrade pip' command.
##[error]The process '/usr/bin/sudo' failed with exit code 1
```

## To Reproduce
1. Go to https://github.com/ros-tooling/setup-ros/actions?query=event%3Aschedule
2. View Failed Tests

--- a/src/package_manager/pip.ts
+++ b/src/package_manager/pip.ts
@@ -40,7 +40,7 @@ const pip3Packages: string[] = [
 	"mock",
 	"mypy",
 	"nose",
-	"numpy",
+	"numpy==1.18.0",
 	"pep8",
 	"pydocstyle",
 	"pyparsing",

